src/App.tsx

```diff
@@ -58,11 +58,11 @@ const App = () => {
 
   const wallets = useMemo(
     () => [
-      getPhantomWallet(),
-      getSolflareWallet(),
-      getSlopeWallet(),
-      getSolletWallet({ network }),
-      getSolletExtensionWallet({ network }),
+      new PhantomWalletAdapter({ network }),
```

Comment:
Phantom's adapter doesn't take a `network` param.

Reply:
Hunh. Why did TypeScript lie to me?